lib/Doctrine/Common/Cache/FileCache.php

```diff
@@ -124,11 +112,13 @@ public function getExtension()
      */
     protected function getFilename($id)
     {
+        $hash = hash('sha256', $id);
```

Comment:
I wouldn't add a performance penalty to every cache for a chance of a collision that is so low that the likelihood of the world ending it higher. Even if a collision occurs, it doesn't matter. Its cache data, not nuclear weapon software.